detox/e2e/test/smoke_test/messaging.e2e.ts

```diff
@@ -111,6 +111,9 @@ describe('Smoke Test - Messaging', () => {
         // * Verify on reply thread screen
         await ThreadScreen.toBeVisible();
 
+        // * Verify no thread overview while there are no replies
```

Comment:
I had trouble running the whole suite, but was able to run this test in isolation with these changes and pass.